exonum/src/node/consensus.rs

```diff
@@ -578,16 +578,18 @@ impl NodeHandler {
             }
         };
 
-        if !tx.verify() {
-            return;
-        }
+        self.execute_later(InternalRequest::VerifyTx(tx));
+    }
 
+    /// Handles an already verified transaction.
+    pub fn handle_verified_tx(&mut self, tx: &dyn Transaction) {
```

Comment:
I wonder that here transaction should pass by value, but transaction code doesn't support that.